battleship.rb

```diff
@@ -0,0 +1,84 @@
+# require 'pry'
+# attr_accessor :radar, :ocean
+#Generates the board, called ocean
+x_axis = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j']
+y_axis = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']
+@ocean = x_axis.product(y_axis).each do |combo|
```

Comment:
Ah, nevermind, I see how that helps in this situation.

I would usually only use instance variables when dealing with a class. In a non-OOP situation, instead of defining instance variables I would make sure to pass in values as arguments to methods.

So in the case of line 34, I would make sure to pass in a `radar` argument to your `add_ship` method. When it comes time for actually calling the method, make sure to pass in a value for that argument that contains what you currently have stored in `@radar`.